sktime/forecasting/base/adapters/_statsmodels.py

```diff
@@ -123,6 +123,88 @@ def _predict(self, fh, X=None):
         y_pred.name = self._y.name
         return y_pred
 
+    @staticmethod
+    def _extract_conf_int(prediction_results, alpha) -> pd.DataFrame:
+        """Construct confidence interval at specified `alpha` for each timestep.
+
+        Parameters
+        ----------
+        prediction_results : PredictionResults
+            results class for predictions
+        alpha : float
+            level of significance
+
+        Returns
+        -------
+        pd.DataFrame
+            confidence intervals at each timestep
+        """
+        del prediction_results, alpha  # tools like ``vulture`` may complain as unused
+
+        raise NotImplementedError("abstract method")
+
+    def _predict_interval(self, fh, X=None, coverage=0.95):
+        """Compute/return prediction interval forecasts.
+
+        private _predict_interval containing the core logic,
+            called from predict_interval and default _predict_quantiles
+
+        Parameters
+        ----------
+        fh : guaranteed to be ForecastingHorizon
+            The forecasting horizon with the steps ahead to to predict.
+        X : optional (default=None)
+            guaranteed to be of a type in self.get_tag("X_inner_mtype")
+            Exogeneous time series to predict from.
+        coverage : float or list of float, optional (default=0.95)
+           nominal coverage(s) of predictive interval(s)
+
+        Returns
+        -------
+        pred_int : pd.DataFrame
+            Column has multi-index: first level is variable name from y in fit,
+                second level coverage fractions for which intervals were computed.
+                    in the same order as in input `coverage`.
+                Third level is string "lower" or "upper", for lower/upper interval end.
+            Row index is fh, with additional (upper) levels equal to instance levels,
+                from y seen in fit, if y_inner_mtype is Panel or Hierarchical.
+            Entries are forecasts of lower/upper interval end,
+                for var in col index, at nominal coverage in second col index,
+                lower/upper depending on third col index, for the row index.
+                Upper/lower interval end forecasts are equivalent to
+                quantile forecasts at alpha = 0.5 - c/2, 0.5 + c/2 for c in coverage.
+        """
+        if not self.get_tag("capability:pred_int"):
```

Comment:
why do you have this in the adapter? This is already in `BaseForecaster.predict_interval`.

Reply:
I checked in `BaseForecaster._predict_interval` by mistake, and found no check. Removed.